corehq/pillows/mappings/case_search_mapping.py

```diff
@@ -52,6 +52,10 @@
                         "numeric": {
                             "ignore_malformed": True,
                             "type": "double"
+                        },
+                        "phonetic": {
```

Comment:
putting this here will analayze every single case property using this analyzer which will likely blow up the size of the index (though we can test it on staging).

Before we roll this out more permanently we must evaluate whether this is an issue we need to address.

Reply:
I'll make a note to keep an eye on this while it's being tested on staging. The goal is to have as little impact as possible if this gets rolled out on prod.

Reply:
Any update on this question?

Reply:
I will look into this further and get some discussion going that addresses this issue specifically. There was some discussion about limiting the number of properties that this is used for, but nothing definitive yet.